src/components/global/Column.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { Droppable } from "react-beautiful-dnd";
+import BoardTask from "./Task/BoardTask";
+
+function Column(props) {
+  const { column, tasks, subTasks, handleList } = props;
+
+  const [open, setOpened] = React.useState(false);
+
+  const onButtonClick = () => {
+    setOpened(!open);
+  };
+
+  return (
+    <div className="column__container">
```

Comment:
hey sorry I wasn't clear about how we were doing it

We only have double underscore for nested components in the div

So the className for the top-level div would be "column-container" and the nested components would be "column-container__heading" or "column-container__sorting", "column-container__sorting-button"

The double underscore is only used once. This helps in organising the scss file which you can see my other comment for